### PR TITLE
Uploaded ching.jpg and edited students.yml

### PR DESCRIPTION
**Pull request by ChingHannie**

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://ChingHannie.github.io/gci15.fossasia.org

File: _data/students.yml [hunk 1]
```diff
@@ -282,4 +282,11 @@
...
   twitter: JLeow00
-  bio: Ukulele is love, ukulele is life
\ No newline at end of file
+  bio: Ukulele is love, ukulele is life
+  
+- name: Ching Hannie
+  github: ChingHannie
+  image: ching.jpg
+  facebook: https://www.facebook.com/abchingh
+  twitter: @hannniieeeee
```
Comment (roonyh) [inline _data/students.yml]: Remove the `@`

File: _data/students.yml [hunk 2]
```diff
@@ -282,4 +282,11 @@
...
   facebook: jerome.leow.5
   twitter: JLeow00
-  bio: Ukulele is love, ukulele is life
\ No newline at end of file
+  bio: Ukulele is love, ukulele is life
+  
+- name: Ching Hannie
+  github: ChingHannie
+  image: ching.jpg
+  facebook: https://www.facebook.com/abchingh
```
Comment (roonyh) [inline _data/students.yml]: We don't need the whole url. Just your username would be enough. Look at other entries in the file.

**Comment by roonyh** [comment]
Please fix the small issues and this should be good to go! Thanks :)

**Comment by roonyh** [comment]
Good! Now take a look at https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md and resolve conflicts. Comment here if something is unclear.

**Comment by mananwason** [comment]
@ChingHannie Please squash all commits into one also. You can find instructions for squashing here https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md

**Comment by jigyasa-grover** [comment]
@ChingHannie Resolve merge conflicts too.

**Comment by ChingHannie** [comment]
@mananwason I have squashed all commits

**Comment by mananwason** [comment]
@ChingHannie The branch still has merge conflicts. Please solve them. Then we can merge !!

**Pull request closed without merging**